Tensile/Common.py

```diff
@@ -158,6 +158,14 @@
 globalParameters["DebugKernel"] = False           # assembly only, kernel gets buffer for debug "printing"; kernel writes data to memory, gets coppied to host and printed
 globalParameters["LibraryPrintDebug"] = False     # solutions will print enqueue info when enqueueing a kernel
 
+# debug for assembly
+globalParameters["EnableAsserts"] = False         # Enable assembly debug assert
+globalParameters["EnableDebugA"] = False          # Enable / Disable CheckValue1A
+globalParameters["EnableDebugB"] = False          # Enable / Disable CheckValue1B
+globalParameters["EnableDebugC"] = False          # Enable / Disable CheckValueC
+globalParameters["ExpectedValueC"] = 16.0         # Expected C Value when CheckValueC, debug for Alpha*A*B
```

Comment:
ExpectedValue = arbitrary?  Not entirely sure this should live in globalParams.

Reply:
This is a quick setter for the debug value in KernelAssemblyWriter :: db["ValueCExpectedValue"] which has been existing from very beginning. I added this block (6 parameters) to help us quickly set the debug value for different input, otherwise we need to modify the KernelAssemblyWriter :: db["...."] value each time. 
Do you have any suggestion to refine this ? Or maybe change the name to "DebugExpectedValueC" to clarify this ?

Reply:
Can probably leave as is for now